api/inc/box_config.h

```diff
@@ -103,6 +105,12 @@ UVISOR_EXTERN const uint32_t __uvisor_mode;
 #define UVISOR_BOX_NAMESPACE(box_namespace) \
     static const char *const __uvisor_box_namespace = box_namespace
 
+/* Use this macro to register privileged system IRQ hooks. Only the main box
+ * can register privileged system IRQ hooks. If you don't want to register any
+ * privileged system IRQ hooks, use this macro with priv_sys_irq_hooks as NULL.
+ * */
+#define UVISOR_SET_PRIV_SYS_IRQ_HOOKS(priv_sys_irq_hooks) \
```

Comment:
Yes, same as UVISOR_BOX_NAMESPACE. This avoids variadic option-parameter macro hell.